geometry/optimization/graph_of_convex_sets.cc

```diff
@@ -377,7 +377,15 @@ std::string GraphOfConvexSets::GetGraphvizString(
     graphviz << " [label=\"" << e->name();
     if (result) {
       graphviz << "\n";
-      graphviz << "cost = " << e->GetSolutionCost(*result);
+      if (e->ell_.size() > 0) {
+        // SolveConvexRestriction does not yet return the rewritten costs.
+        if (result->get_decision_variable_index()->count(e->ell_[0].get_id()) !=
```

Comment:
Why is the line on 382 necessary? 

GetSolutionCost sums the `ell_` vector in line 238, I think that summation will succeed if `ell_` is not size 0?